### ISSUE TITLE
remove "project" facet from general search

### ISSUE THREAD

**Issue by jrochkind** [sciencehistory/scihist_digicoll#1162]
As it's only meant to be used for Oral Histories at present, so shows up as kind of a barely populated facet in general search. And now we have OH-speciifc search that has it. 

However, doing this requires some changes to Work display too. Right now, any Work with one or more "projects" assigned has that show up in it's metadata, with the projects being hyperinks that send you to a general search limited to that project in project facet -- if project facet isn't on general search, it can't do this. 

It could instead direct to that facet limit in the Oral History-specific search... but this maybe starts to get confusing. 

OR, we could leave it alone, and say, yeah, it makes the facets a bit confusing in general search, but we aren't sure how much use facets are getting anyway or what their biggest UX problems are, so since this change got somewhat more complicated, just put it on the backburner for now. 

Thoughts @apinkney0696  or @MDiMeo ?

**Comment by apinkney0696** [comment]
Hm, I see. I like the ability to link the projects through general search, but it may be sufficient for researcher's purposes to only have it as a refinement option only on the OH-specific search. I think it would be worth knowing how often our users are using this as a general search feature in real life before making a decision. 

I'm not sure if this is something I can pull data for from Google Analytics, but I can do a little digging at some point if you think it would be worth it.

**Comment by jrochkind** [comment]
In my memory we already had agreement to remove the 'project' facet from general search (it will remain on OH-specific search) -- the issue is that when I went to do so, I realized it interacted with the *display* of "project" values on OH (as links), such that it required more conversation. 

As far as the case for removing it from the general search -- for me it's that, since "project" only applies to OH records (and not even to all of them), then in any given search, only a minority of records in the results are likely to have a "project" assigned. 

Almost all of the other facets we have, *all*, or at least most, records will have *some* value. I think this is the way facets are intended to work. 

When we already have so many facets -- and I suspect they are already overwhelming -- adding yet another one that only a very small subset even have values for, I think just adds to the "noise" of facets for users. 

For instance, I'm a user [searching for "Alchemy"](https://digital.sciencehistory.org/catalog?search_field=all_fields&q=alchemy&range%5Byear_facet_isim%5D%5Bbegin%5D=&range%5Byear_facet_isim%5D%5Bend%5D=), and then open up the "projects" facet. I am a user -- do I know what this even means, how is it of use to me, what's the use case where this is helpful?  I think it just adds to the confusion of the facets interface, which I suspect users already find overwhelming and confusing, in part because we have so many.
